crates/nbformat/tests/conformance.rs

```diff
@@ -398,7 +398,7 @@ mod test {
      "data": {
       "image/png": [
        "iVBORw0KGgoAAAANSUhEUgAAAAoAAAAKCAYAAACNMs+9AAAAFUlEQVR42mNk+M9Qz0AEYBxVSF+F\n",
-       "AAh KDveksOjmAAAAAElFTkSuQmCC\n"
+       "AAhKDveksOjmAAAAAElFTkSuQmCC\n"
```

Comment:
This was a mistake while I was playing around with this one. Now it tests and creates the right image data for comparison.